src/components/Dialog/DialogContent/DialogContent.tsx

```diff
@@ -65,6 +70,18 @@ export const DialogContent: VibeComponent<DialogContentProps> = React.forwardRef
     );
     useKeyEvent({ keys: ESCAPE_KEYS, callback: onEsc });
     useOnClickOutside({ callback: onOutSideClick, ref });
+    const selectorToDisable = typeof disableContainerScroll === "string" ? disableContainerScroll : containerSelector;
+    const { disableScroll, enableScroll } = useDisableScroll(selectorToDisable);
+
+    useEffect(() => {
+      if (disableContainerScroll) {
+        if (isOpen) {
+          disableScroll();
+        } else {
+          enableScroll();
+        }
+      }
+    }, [isOpen]);
```

Comment:
Should add also a cleanup when component is I mounted (the return function) and add a dependency to the hook given the disableContainerScroll